packages/react/test/specs/components/Attachment/Attachment-test.tsx

```diff
@@ -18,5 +21,30 @@ describe('Attachment', () => {
     handlesAccessibility(Attachment, {
       defaultRootRole: undefined,
     })
+
+    test('handleClick is executed when Enter is pressed for attachment, not for more option button', () => {
+      const onClickAttachment = jest.fn()
+      const onClickButton = jest.fn()
+      const attachment = mountWithProvider(
+        <Attachment
+          actionable
+          action={{
+            icon: 'more',
+            onClick: () => onClickButton(),
```

Comment:
can be `onClickButton` as the similar one below

Reply:
changed :)